### PR TITLE
Add build and test instructions to README.md

### PR DESCRIPTION
**Pull request by jtattermusch**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -27,7 +27,19 @@ Instructions coming soon!
 
 ## To develop gRPC for ASP.NET Core
 
-TODO: add development instructions
+Install [.NET Core SDK 3 preview 2](https://dotnet.microsoft.com/download/dotnet-core/3.0)
+or [VS 2019 preview](https://visualstudio.microsoft.com/vs/preview/).
```
Comment (JamesNK) [inline README.md]: I'm pretty sure VS2019 doesn't ship with .NET Core 3 preview 2.

Comment (jtattermusch) [reply README.md]: Ok, so there's currently no way to build with VS? (I haven't tried yet).

Comment (JamesNK) [reply README.md]: VS2017 or VS2019 should work at the moment once you have downloaded preview 2.

I'm not sure about VS Code, I don't use it myself. @davidfowl are you using VS Code?

Comment (davidfowl) [reply README.md]: I'm using VS 2019

**Comment by davidfowl** [comment]
I just noticed the Experimental Only section in the README. Can we change that to say something to the effect that's it's pre-release instead of experimental?

**Comment by jtattermusch** [comment]
@davidfowl  well right now it's really just experimental, you can't really do anything useful with it - so I think that characterization is accurate. But feel free to propose alternative wording - I just wanted to warn potential visitors that this doesn't work just yet.

**Comment by davidfowl** [comment]
I disagree. Experimental means that we don't know if this is going to work not, not that it's going to ship. We use that experimental term for actual experiments that we're trying to figure out if it'll work out. I'd rather say something like this is under active development and there's no release for it as yet. That's much more the reality of the situation rather than calling it an experiment.

**Comment by jtattermusch** [comment]
@davidfowl  makes sense, can you propose the right wording then? Alternatively, feel free to create a PR to update yourself.

**Comment by JunTaoLuo** [comment]
I tried updating the comments over at https://github.com/jtattermusch/grpc-dotnet/pull/1. I don't think I can push to your branch so I opened a PR. @JamesNK @davidfowl @jtattermusch PTAL.

**Comment by jtattermusch** [comment]
Integrated John's suggestions into this PR.

**Pull request merged**